### DISCUSSION TITLE
Videos not working

### DISCUSSION THREAD

**Discussion by gamarrita** [Mair/esp32-course#123] [Course Qeustions]
Since yesterday, 06/22 can't play any video, when pressing play keeps loading forever

**Comment by Mair** [comment]
Sorry for the delayed response.  Could I please ask you to clear your cache and let me know if its still al issue? 
failing that could you please have a look at https://www.thewindowsclub.com/vimeo-not-working-chrome-firefox and let me know if that helps.

**Reply by Mair** [reply]
@gamarrita Is this OK now?
